src/docGeneratorMain.ts

```diff
@@ -83,305 +76,280 @@ async function main() {
     false
   )
 
   console.log("Generate i18next description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/edvardchen/eslint-plugin-i18next/next/docs/rules/`,
     "i18next",
     false
   )
 
   console.log("Generate import description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/un-es/eslint-plugin-i/fork-release/docs/rules/`,
     "import",
     false
   )
 
   console.log("Generate jasmine description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/tlvince/eslint-plugin-jasmine/master/docs/rules/`,
     "jasmine"
   )
 
   console.log("Generate jest description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/jest-community/eslint-plugin-jest/master/docs/rules/`,
     "jest"
   )
 
   console.log("Generate jest-dom description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/testing-library/eslint-plugin-jest-dom/main/docs/rules/`,
     "jest-dom"
   )
 
   console.log("Generate jest-extended description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/jest-community/eslint-plugin-jest-extended/main/docs/rules/`,
     "jest-extended"
   )
 
   console.log("Generate jest-formating description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/dangreenisrael/eslint-plugin-jest-formatting/master/docs/rules/`,
     "jest-formating"
   )
 
   console.log("Generate jsdoc description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/gajus/eslint-plugin-jsdoc/main/docs/rules/`,
     "jsdoc"
   )
 
   console.log("Generate jsonc description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/ota-meshi/eslint-plugin-jsonc/master/docs/rules/`,
     "jsonc"
   )
 
   console.log("Generate jsx-a11y description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/jsx-eslint/eslint-plugin-jsx-a11y/main/docs/rules/`,
     "jsx-a11y"
   )
 
   console.log("Generate lit description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/43081j/eslint-plugin-lit/master/docs/rules/`,
     "lit"
   )
 
   console.log("Generate lodash description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/wix-incubator/eslint-plugin-lodash/master/docs/rules/`,
     "lodash"
   )
 
   console.log("Generate lodash-fp description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/jfmengels/eslint-plugin-lodash-fp/master/docs/rules/`,
     "lodash-fp"
   )
 
   console.log("Generate meteor description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/dferber90/eslint-plugin-meteor/master/docs/rules/`,
     "meteor"
   )
 
   console.log("Generate mocha description files")
   await docGenerator.downloadDocs(
-
     `${githubBaseUrl}/lo1tuma/eslint-plugin-mocha/master/docs/rules/`,
     "mocha"
   )
 
-  console.log("Generate no-unsanitized description files")
+  console.log("Generate n description files")
```

Comment:
The "no-unsanitized" is kept.
Changed "node" to "n" because:
"forked from [eslint-plugin-node](https://www.npmjs.com/package/eslint-plugin-node) v11.1.0. as the original repository seems [no longer maintained](https://github.com/mysticatea/eslint-plugin-node/issues/300)."